spec/takeaway_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'takeaway'
+require 'timecop'
+
+describe TakeAway do
+  let(:menu) { double(:menu) }
+  before(:each) do
+    allow(menu).to receive(:dishes).and_return("chilli squid" => 3.95, "pork wonton" => 2.95)
+  end
+
+  it "can add a menu" do
+    expect(subject.add_menu(menu)).to eq(menu.dishes)
+  end
+
+  describe "#place_order" do
+    before(:each) do
+      subject.add_menu(menu)
+    end
+
+    it "adds item to order hash" do
+      subject.place_order("chilli squid" => 1)
+      expect(subject.order).to include("chilli squid")
+    end
+
+    it "adds correct quantity of item to order" do
+        subject.place_order("chilli squid" => 3)
+        expect(subject.order["chilli squid"]).to eq(3)
+      end
```

Comment:
`end` at 27, 6 is not aligned with `it "adds correct quantity of item to order" do` at 24, 4